src/re_frame/context.clj

```diff
@@ -14,12 +14,15 @@
        ^{:context-type frame-context}
        (fn ~@fntail))))
 
-(defmacro import-with-frame [var-sym]
-  `(defn
-     ~(symbol (name var-sym))
-     ;; Attempt at propagating the doc string / arglists, for some reason CIDER
-     ;; is not picking this up though.
-     ~(select-keys (:meta (cljs.analyzer/resolve-var cljs.env/*compiler* var-sym))
-                   [:doc :arglists])
-     [& args#]
-     (apply ~var-sym (current-frame) args#)))
+(defmacro import-with-frame
+  ([var-sym]
+   `(import-with-frame ~(symbol (name var-sym)) ~var-sym))
+  ([name var-sym]
+   `(defn
+      ~(symbol (name var-sym))
```

Comment:
This should be `~name` instead? `name` is shadowing `cljs.core/name` here so this is not going to work as is I think.